### PR TITLE
Projections value must be selectable when storage type selected as journal.

### PR DESCRIPTION
**Pull request by hurelhuyag**

### CODE REVIEW

**Comment by Florian-Schoenherr** [comment]
with your solution, try selecting another storage type (not Journal), then select 'Operation Based', now select Journal - if it is still 'Operation Based', that's wrong. So you would need to check (if) that and set it to 'None' accordingly.

**Comment by hurelhuyag** [comment]
@Florian-Schoenherr Maybe I'm wrong. I'm not svelte developer. All I can say is that without this change I can't select 'Journal' with 'Event Based'

**Comment by hurelhuyag** [comment]
I discovered that `changedStorageType` function runs every time I click `Persistence` menu. That is how `Projections` value is cleared. And I updated my code. I think this is better solution.

File: src/main/frontend/src/routes/persistence.svelte [hunk 1]
```diff
@@ -30,6 +30,9 @@
 				{name: 'Not Applicable', value: 'NONE'},
 				{name: 'Event Based', value: 'EVENT_BASED'},
 			];
+			if($projectionsTypes.filter(p => p.value === projections).length < 1){
```
Comment (Florian-Schoenherr) [inline src/main/frontend/src/routes/persistence.svelte:33]:
```suggestion
			if(projections === 'OPERATION_BASED') {
```

**Review by Florian-Schoenherr — CHANGES_REQUESTED** [review]
This is what I meant, can we merge after that change?
Should work for you :)

File: src/main/frontend/src/routes/persistence.svelte (hunk 1 above)
Comment (hurelhuyag) [reply src/main/frontend/src/routes/persistence.svelte:33]: Yes this will work if "projectionsTypes" will not change in the future.

**Review by Florian-Schoenherr — APPROVED** [review]

**Pull request merged**